tests/System/TextPolicyValidatorTest.php

```diff
@@ -69,14 +71,28 @@ public function testWhenGivenHarmlessComment_validatorReturnsTrue( $commentToTes
 
 	public function harmlessTestProvider() {
 		return [
-			[ 'Ich mag Wikipedia.Wieso ? Weil ich es so toll finde!' ],
 			[ 'Wikipedia ist so super, meine Eltern sagen es ist eine toll Seite. Berlin ist auch Super.' ],
 			[ 'Ich mag Wikipedia. Aber meine Seite ist auch toll. Googelt mal nach Bunsenbrenner!!!1' ],
 			[ 'Bei Wikipedia kann man eine Menge zum Thema Hamster finden. Hamster fressen voll viel Zeug alter!' ],
 			[ 'Manche Seiten haben keinen Inhalt, das finde ich sch...e' ], // this also tests the domain detection
 		];
 	}
 
+	public function testHamlessContentWithDns() {
+		$this->skipIfNoInternet();
+
+		if ( checkdnsrr( 'some-non-existing-domain-drfeszrfdaesr.sdferdyerdhgty', 'A' ) ) {
+			$this->markTestSkipped( 'Sure, blame it on your ISP!' );
```

Comment:
I'd rather have a more informative message like "Your DNS/ISP provider gives results for impossible host names."